system-tests/src/tests/draft.spec.ts

```diff
@@ -0,0 +1,109 @@
+import { expect, test } from "@playwright/test"
+
+import expectScreenshotsToMatchSnapshots from "../utils/screenshot"
+
+test.describe("anonymous user", () => {
+  test("cannot see draft course", async ({ page }) => {
+    // Go to http://project-331.local/
+    await page.goto("http://project-331.local/")
+
+    // Click text=University of Helsinki, Department of Mathematics and Statistics
+    await Promise.all([
+      page.waitForNavigation(/*{ url: 'http://project-331.local/org/uh-mathstat' }*/),
+      page.click("text=University of Helsinki, Department of Mathematics and Statistics"),
+    ])
+
+    await expect(page.locator("text=Introduction to Statistics")).toBeVisible()
+    await expect(page.locator("text=Introduction to Drafts")).not.toBeVisible()
+  })
+})
+
+test.describe("user", () => {
```

Comment:
Also tests where the not authorized user would try to go directly to the url and get a not found message